src/js/jsx/shared/Datalist.jsx

```diff
@@ -527,7 +560,8 @@ define(function (require, exports, module) {
             }
             
             var autocomplete,
-                hiddenTI;
+                hiddenTI,
+                hiddenSVG;
  
             if (this.props.useAutofill) {
                 hiddenTI = (
```

Comment:
Can you leave a few `// comments` here that describe the point of these hidden elements?